compiler/src/dmd/backend/cc.d

```diff
@@ -1190,6 +1190,7 @@ enum
     SFLweak         = 0x1000000,   // resolve to NULL if not found
     SFLhidden       = 0x2000000,   // not visible outside of DSOs (-fvisibility=hidden)
     SFLartifical    = 0x4000000,   // compiler generated symbol
+    SFLnounderscore = 0x8000_0000, // don't prepend an _ to identifiers in object file
```

Comment:
Underscores may be what x86 Windows and OSX do, but there are other forms of abi symbol prefixing, not just underscore.